public/app/percona/inventory/Tabs/Services.tsx

```diff
@@ -93,24 +95,36 @@ export const Services = () => {
     setSelectedRows(rows);
   }, []);
 
+  const handleAddService = useCallback(() => {
+    locationService.push('/add-instance');
+  }, []);
+
   return (
     <OldPage navModel={navModel}>
       <OldPage.Contents>
         <FeatureLoader>
           <div className={styles.tableWrapper}>
-            <div className={styles.actionPanel}>
-              <Button
-                size="md"
-                disabled={selected.length === 0}
-                onClick={() => {
-                  setModalVisible(!modalVisible);
-                }}
-                icon="trash-alt"
-                variant="destructive"
-                className={styles.destructiveButton}
-              >
-                Delete
-              </Button>
+            <div className={styles.servicesActionPanel}>
+              <HorizontalGroup justify="space-between">
+                {selected.length === 0 ? (
+                  <div className={styles.description}>Select services to bulk edit them</div>
+                ) : (
+                  <Button
+                    size="md"
+                    onClick={() => {
+                      setModalVisible((visible) => !visible);
+                    }}
+                    icon="trash-alt"
+                    variant="destructive"
+                    className={styles.destructiveButton}
+                  >
+                    {selected.length === 1 ? `Delete ${selected.length} service` : `Delete ${selected.length} services`}
+                  </Button>
+                )}
+                <Button size="md" onClick={handleAddService} icon="plus" variant="primary">
+                  Add service
```

Comment:
Can you move this text to message file? Also I noticed that all texts in this folder are not in separate file. Can you move them as well since you are editing this?